app/controllers/membership_applications_controller.rb

```diff
@@ -45,11 +45,7 @@ def update
 
       new_upload_file params['uploaded_file'] if params['uploaded_file']
 
-      if changed_to_accepted?(params['membership_application'])
-        accept_application
-        redirect_to edit_membership_application_url(@membership_application)
-        return
-      end
+      @membership_application.applicant_updated_info! if (current_user.id == @membership_application.user.id)
```

Comment:
We also have to test that if a user edits their application and adds a category, submits it for review, the new category is not public until approved... 
I don't know if our system caters for this?